lib/galaxy/dependencies/conditional-requirements.txt

```
@@ -56,7 +56,12 @@ tensorflow==2.9.3
 # xref https://github.com/galaxyproject/galaxy/pull/9651
 # Run run.sh or common_startup script with GALAXY_DEPENDENCIES_INSTALL_WEASYPRINT=1
 # to install weasyprint as part of Galaxy's conditonal dependency instalation process.
-weasyprint
+weasyprint>=61.2
+# We pin the transitive weasyprint dependency pydyf here because newer versions are
+# not compatible with the last weasyprint version that still supports python 3.8.
+# This dependency is ignored on python >= 3.9 and should be removed when deprecating
+# support for python 3.8.
+pydyf<0.12; python_version<"3.9"
```

Ah, I thought the problem was 0.12.0, but it's 0.11.0 !

```suggestion
pydyf<0.11; python_version<"3.9"
```